cmake/cable/buildinfo/buildinfo.h.in

```diff
@@ -26,7 +26,7 @@ struct buildinfo
     const char* build_type;
 };
 
-const struct buildinfo* @FUNCTION_NAME@();
+const struct buildinfo* @FUNCTION_NAME@(void);
```

Comment:
Put changes to cable in separate commit and possibly add a CHANGELOG entry in the cmake file.

Reply:
done